src/fileManagement.js

```diff
@@ -218,6 +217,7 @@ files.prototype.delete = function (path) {
 /**
  * Returns the file info for the given remote file
  * @param   {string}  path          path of the file/folder at OC instance
+ * @param   {array}   properties    WebDAV properties
```

Comment:
is this an array of the property names `{Array.<string>}` ? or a hash map of property name to values `{Object.<string, string>}` ?